src/main/java/org/mule/extension/api/introspection/ExceptionHandled.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.introspection;
+
+import java.util.Optional;
+
+/**
+ * Contract for models capable of providing an optional {@link ExceptionEnricherFactory}
+ *
+ * @see Interceptable
+ * @since 1.0
+ */
+public interface ExceptionHandled
+{
+
+    /**
+     * Returns an optional concrete type of the {@link ExceptionEnricherFactory}.
```

Comment:
This sentence doesn't actually describe what the method does. It's also unnecesary given what you wrote at the @return